getting_started/step_by_step/your_first_game.rst

```diff
@@ -168,6 +168,9 @@ click "Create":
 
 .. note:: If this is your first time encountering GDScript, please read
           :ref:`doc_scripting` before continuing.
+          
+.. note:: If this is your first time encountering C# in Godot, please read
+          `C# basics <https://docs.godotengine.org/en/stable/getting_started/scripting/c_sharp/c_sharp_basics.html>`_ before continuing.
```

Comment:
Use an internal reference rather than an external one. This way, the version will match the current documentation version and language automatically:

```suggestion
          :ref:`doc_c_sharp` before continuing.
```

Reply:
Thanks, I updated as such in the latest commit. I actually wanted to use an internal ref from the beginning but I didn't know what name did I need to use, so resorted to an external one :sweat_smile: Is there any document that lists such internal references?

Reply:
Does ``:ref:`doc_c_sharp` `` reference to [C# basics](https://docs.godotengine.org/en/stable/getting_started/scripting/c_sharp/c_sharp_basics.html) or to the [C# index](https://docs.godotengine.org/en/stable/getting_started/scripting/c_sharp/index.html) page?
My proposal was to have the link pointing to the [C# basics](https://docs.godotengine.org/en/stable/getting_started/scripting/c_sharp/c_sharp_basics.html) as it contains details on how to set up C# in Godot, which I find useful to have a smoother experience going forward